### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by VxlerieUwU**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,1428 @@
...
+msgid "Call and text notification"
+msgstr "Notifications d'appel et de SMS"
+
+#: data/metainfo/ca.andyholmes.Valent.metainfo.xml.in.in:28
+msgid "Execute custom commands"
+<<<<<<< HEAD
+msgstr "Éxecuter des commandes personnalisées"
+=======
+msgstr "Exécuter des commandes personnalisées"
+>>>>>>> d9df43a (Added french translation)
```
Comment (andyholmes) [inline po/fr.po:101]:
```suggestion
<<<<<<< HEAD
msgstr "Éxecuter des commandes personnalisées"
=======
msgstr "Exécuter des commandes personnalisées"
>>>>>>> d9df43a (Added french translation)
```

Looks like this got caught up in a rebase. I'm not sure which is correct though, it's been years :slightly_smiling_face:

File: po/fr.po [hunk 2]
```diff
@@ -0,0 +1,1428 @@
...
+msgid "Commands"
+msgstr "Commandes"
+
+#: src/plugins/runcommand/runcommand.plugin.desktop.in:8
+msgid "Execute commands remotely"
+<<<<<<< HEAD
+msgstr "Éxecuter des commandes à distance"
+=======
+msgstr "Exécuter des commandes à distance"
+>>>>>>> d9df43a (Added french translation)
```
Comment (andyholmes) [inline po/fr.po:948]: Also here.

**Review by andyholmes — CHANGES_REQUESTED** [review]
LGTM, except for the conflict markers. Thanks!

File: po/fr.po (hunk 2 above)
Comment (VxlerieUwU) [reply po/fr.po]: Both are done

**Comment by VxlerieUwU** [comment]
Thanks for letting me know! Everything should be cleaned up now

**Review by andyholmes — APPROVED** [review]

**Pull request merged**